assets/scss/00-base/_fonts.scss

```diff
@@ -132,3 +132,171 @@
+  font-style: normal;
+  font-weight: 600;
+  src: url('../fonts/source-sans-pro-600.eot'); // IE9 Compat Modes.
+  src: local('Source Sans Pro SemiBold'), local('SourceSansPro-SemiBold'),
+  url('../fonts/source-sans-pro-600.eot?#iefix') format('embedded-opentype'), // IE6-IE8.
+  url('../fonts/source-sans-pro-600.woff2') format('woff2'), // Super Modern Browsers.
+  url('../fonts/source-sans-pro-600.woff') format('woff'), // Modern Browsers.
+  url('../fonts/source-sans-pro-600.ttf') format('truetype'), // Safari, Android, iOS.
+  url('../fonts/source-sans-pro-600.svg#SourceSansPro') format('svg'); // Legacy iOS.
+}
+
+// source-sans-pro-600italic - latin.
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: italic;
+  font-weight: 600;
+  src: url('../fonts/source-sans-pro-600italic.eot'); // IE9 Compat Modes.
+  src: local('Source Sans Pro SemiBold Italic'), local('SourceSansPro-SemiBoldItalic'),
+  url('../fonts/source-sans-pro-600italic.eot?#iefix') format('embedded-opentype'), // IE6-IE8.
+  url('../fonts/source-sans-pro-600italic.woff2') format('woff2'), // Super Modern Browsers.
+  url('../fonts/source-sans-pro-600italic.woff') format('woff'), // Modern Browsers.
+  url('../fonts/source-sans-pro-600italic.ttf') format('truetype'), // Safari, Android, iOS.
+  url('../fonts/source-sans-pro-600italic.svg#SourceSansPro') format('svg'); // Legacy iOS.
+}
+
+// source-sans-pro-700 - latin.
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 700;
+  src: url('../fonts/source-sans-pro-700.eot'); // IE9 Compat Modes.
+  src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'),
+  url('../fonts/source-sans-pro-700.eot?#iefix') format('embedded-opentype'), // IE6-IE8.
+  url('../fonts/source-sans-pro-700.woff2') format('woff2'), // Super Modern Browsers.
+  url('../fonts/source-sans-pro-700.woff') format('woff'), // Modern Browsers.
+  url('../fonts/source-sans-pro-700.ttf') format('truetype'), // Safari, Android, iOS.
+  url('../fonts/source-sans-pro-700.svg#SourceSansPro') format('svg'); // Legacy iOS.
+}
+
+// source-sans-pro-700italic - latin.
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: italic;
+  font-weight: 700;
+  src: url('../fonts/source-sans-pro-700italic.eot'); // IE9 Compat Modes.
+  src: local('Source Sans Pro Bold Italic'), local('SourceSansPro-BoldItalic'),
+  url('../fonts/source-sans-pro-700italic.eot?#iefix') format('embedded-opentype'), // IE6-IE8.
+  url('../fonts/source-sans-pro-700italic.woff2') format('woff2'), // Super Modern Browsers.
+  url('../fonts/source-sans-pro-700italic.woff') format('woff'), // Modern Browsers.
+  url('../fonts/source-sans-pro-700italic.ttf') format('truetype'), // Safari, Android, iOS.
+  url('../fonts/source-sans-pro-700italic.svg#SourceSansPro') format('svg'); // Legacy iOS.
+}
+
+// source-sans-pro-900 - latin.
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 900;

```

Comment:
is there a reason to set font-weight to 900 vs 800? I'm asking because Texta is using 800 for black. I'm also adding a monospace font

Reply:
More than anything, it is because the font files are identified as the 900 weight version of the file. I am unsure what issues could arise from using the 900 weight files, yet designating them as 800. This might introduce weird edge-case bugs. Not sure.